vlib/regex/README.md

```diff
@@ -133,17 +133,18 @@ Suppose you have `abccc ddeef` as a source string, that you want to parse
 with a regex. The following table show the query strings and the result of
 parsing source string.
 
-+--------------+-------------+
 | query string |   result    |
 |--------------|-------------|
 | `.*c`        | `abc`       |
 | `.*dd`	   | `abcc dd`   |
 | `ab.*e`      | `abccc dde` |
 | `ab.{3} .*e` | `abccc dde` |
-+--------------+-------------+
-
 The dot matches any character, until the next token match is satisfied.
 
+**Important Note:** *multiple consecutive like `..` are not allowed, it is managed as syntax error.*
```

Comment:
It differentiates from "multiple non-consecutive".  However, I would state it more simply as
```
Consecutive dots, for example `...`, are not allowed.  This will cause a syntax error.  Use a quantifier instead.
```
This covers your other comments as well.  :-)